src/Azure.Functions.Cli/ArtifactsConfigurationBuilder.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Azure.Functions.Cli.Common;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Functions.Cli
+{
+    internal class ArtifactsConfigurationBuilder : IConfigureBuilder<IConfigurationBuilder>
```

Comment:
Is your goal with this class just to assist with registering that `builder.AddJsonFile` call on line 16? If so, this can be accomplished without needing to use an interface / instantiate this type.

Option 1: extension method
``` CSharp
internal static class ConfigurationBuilderExtensions
{
    public static IConfigurationBuilder AddArtifactsConfigFile(this IConfigurationBuilder builder)
    {
        return builder.AddJsonFile(Constants.ArtifactsConfigFileName, optional: true);   
    }
}
```

Option 2: Just inline it

This is only used once, so it wouldn't be the worst thing to just put `builder.AddJsonFile(Constants.ArtifactsConfigFileName, optional: true);` directly where it is used.

Reply:
Inline version looks good